### PR TITLE
journals page made responsive, align journals form and heading in the center

### PR DESCRIPTION
**Pull request by vedeepika**
### Issues Fixed

1). The form and heading of the journal page were not in the center. I aligned the form and heading in the center. 

Screenshot (54)

2). I Made the Journals page responsive on all the devices.

Screenshot (55)

## Type of change

- [x] New feature (non-breaking change which adds functionality).

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding tests.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [x] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

File: src/pages/Journal.js [hunk 1]
```diff
@@ -7,8 +7,13 @@ import "../styles/Journal.css"
 import AddJournal from '../components/AddJournal';
 
 const Journal = () => (
+
+
+
+
```
Comment (padmajabhol) [inline src/pages/Journal.js]: I don't think we need the white spaces here.

Comment (padmajabhol) [inline src/pages/Journal.js]: I don't think we need the white spaces here.

File: src/styles/Login.css [hunk 2]
```diff
@@ -9,6 +9,15 @@
   margin-top: 50px;
   margin-bottom: 50px;
 }
+
```
Comment (padmajabhol) [inline src/styles/Login.css]: Remove the extra white-space

**Review by padmajabhol — CHANGES_REQUESTED** [review]

**Review by vedeepika — COMMENTED** [review]
Removed whitespace.

**Review by padmajabhol — APPROVED** [review]

**Pull request merged**